docker-images/kafka/cruise-control-scripts/cruise_control_healthcheck.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+SCHEME="http"
+HOST="$(hostname | rev | cut -d "-" -f3- | rev):${API_PORT}"
```

Comment:
Why not just use localhost / 127.0.0.1? the healthcheck should always run locally.

Reply:
The `--resolve "${HOST}:127.0.0.1"`arg to curl resolves the host address to `0.0.0.0` (I will update 0.0.0.0 to 127.0.0.1 in next commit). I had to do it this way to get the  TLS hostname verification working using localhost/127.0.0.1 since the Common Name of the CC public certificate lists the hostname of the pod

Reply:
Why don't you just do 127.0.0.1 and disable hostname verification?

Reply:
I guess I could do this, this is what k8s HTTPS probes do anyway and it would simplify the script. From what I understand it shouldn't be much of a risk if I am using the 127.0.0.1 address

Reply:
After adding the appropriate domains to the Cruise Control certificate, the `resolve` argument is now unnecessary. Now we have certificate verification without the extra scripts